sched/init/nx_start.c

```diff
@@ -38,6 +38,7 @@
 #include <nuttx/mm/iob.h>
 #include <nuttx/mm/mm.h>
 #include <nuttx/kmalloc.h>
+#include <nuttx/pgalloc.h>
```

Comment:
why we need this change?

Reply:
To forward declare `mm_pginitialize`

Reply:
This is the only change in this file, so `mm_pginitialize` isn't reported as undeclared for other platforms?

Reply:
CONFIG_MM_PGALLOC is used by a very few targets only (e.g. sama5d4-ek:knsh), I assume the declaration either gets included via some other file or those configs are not tested by the CI. 

With RISC-V the build fails.